index.js

```diff
@@ -1 +1,118 @@
 // implement your API here
+const express = require('express');
+
+const server = express();
+
+server.use(express.json());
+
+const db = require('./data/db');
+
+// POST to db -------------------------------------
+
+server.post('/api/users', (req, res) => {
+    const dbInfo = req.body;
+
+    db.insert(dbInfo)
+        .then(info => {
+            res.status(201).json(info)
+        })
+        .catch(() => {
+            if (dbInfo !== "name" && "bio") {
```

Comment:
this is kind of excess... if you are going to compare your variable with the content, you might as well save a line of code and just write req.body !== "name" && "bio"
It's an interesting approach